at_contacts_group_flutter/lib/widgets/custom_toast.dart

```diff
@@ -2,32 +2,49 @@ import 'dart:io';
 
 import 'package:at_contacts_group_flutter/utils/colors.dart';
 import 'package:flutter/material.dart';
-import 'package:fluttertoast/fluttertoast.dart';
-import 'package:flutter_toastr/flutter_toastr.dart';
 
 class CustomToast {
   CustomToast._();
   static final CustomToast _instance = CustomToast._();
   factory CustomToast() => _instance;
 
-  // ignore: always_declare_return_types
-  show(String text, BuildContext context,
-      {Color? bgColor, Color? textColor, int duration = 3, int gravity = 0}) {
-    if (Platform.isLinux || Platform.isWindows || Platform.isMacOS) {
-      FlutterToastr.show(
-        text,
-        context,
-        duration: FlutterToastr.lengthLong,
+  /// pass [isError] true to have a red bg, [isSuccess] true to have a green bg.
```

Comment:
@yahu1031 I think we need not add this class in all the packages, it can be reused from at_commons_flutter.

Reply:
yes but we haven't released the package yet